src/App.test.jsx

```diff
@@ -0,0 +1,16 @@
+import { describe, it, expect } from "vitest";
+import { calculateRemainedDays } from "./components/Todo/Item"
+
+describe("calculateRemainedDays", () => {
+  it("Expect to return 10 that means 10 days remained", () => {
+    expect(calculateRemainedDays("2/13/2025")).toBe(10)
+  });
+
+  it("Expect to return -2 that means 2 days passed", () => {
+    expect(calculateRemainedDays("2/1/2025")).toBe(-2)
+  })
```

Comment:
These two are tricky because they were only true on 02/03/2025.

To address this, a `beforeAll` hook can be used to fake the current time. along the lines of this

```
  beforeAll(() => {
    // Mock the current date to a fixed value for predictable test results
    jest.useFakeTimers();
    jest.setSystemTime(new Date('2025-02-03T11:59:59'));
  });
```